server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
```

Comment:
@rfk - have you opened a corresponding puppet-config PR to add this new config value, or is one already set?